### PR TITLE
feature(remove-env): Using Env not needed for react-native

### PR DESCRIPTION
**Pull request by ryanbas21**
Due to the complexity of adding a certificate on mobile, along with the
difficulties of using env files in react native (caching issues) we do
not need an env file for the react native portion. We also may not need
an env file at all if we go with a hosted server and have the user edit
the plist file directly.

https://bugster.forgerock.org/jira/browse/SDKS-1213

### CODE REVIEW

File: reactnative-todo/src/components/utilities/request.js [hunk 1]
```diff
@@ -26,7 +24,8 @@ async function request(method, resource = '', body = null) {
     const json = await FRAuthSampleBridge.getAccessToken();
     const tokens = JSON.parse(json);
     const { tokenType, value } = tokens;
-    const res = await fetch(`${API_URL}/todos/${resource}`, {
+     // edit the url here in fetch with the url for your server
+    const res = await fetch(`https://api.example.com:9443/todos/${resource}`, {
```
Comment (cerebrl) [inline reactnative-todo/src/components/utilities/request.js:28]: Do we have a `constants` file or a better place to put such a value. I'm afraid this is a bit hidden. Also, I'd like to have a check when the app boots up to check if a value is present, if not, it errors.

**Review by cerebrl — CHANGES_REQUESTED** [review]
What if we created a `.env.js` file, rather than a `.env` file? We could move all values over to it. `gitignore` the `.env.js` file but have a `.env.js.example` that isn't ignored that they use as a template.

**Comment by daveadams56** [comment]
I agree with Justin about moving the API URL to a .js file within the project. If there is a .env file setup within the project for the backend, perhaps a step in the build that could read .env and inject into a config file within the react native project could reduce duplication? An example where I've done this for the Angular sample app is here: [set-env.mjs](https://github.com/daveadams56/angular-todo-prototype/blob/main/set-env.mjs)

File: reactnative-todo/.env.example.js [hunk 2]
```diff
@@ -0,0 +1 @@
+const API_URL = ''
```
Comment (cerebrl) [inline reactnative-todo/.env.example.js:1]: Could we just move all of the variables from `.env`, even those related to the Node.js server needs?

File: README.md [hunk 3]
```diff
@@ -12,14 +12,14 @@ ForgeRock protected native/hybrid application using React Native and the ForgeRo
 6. `cd ios`
 7. `pod install`
 8. `cd ..`
-9. Create the `.env` file from the `.env.example` file
+9. Create the `.env.mjs` file from the `.env.example.js` file
```
Comment (cerebrl) [inline README.md:15]: This should be `.env.js`, yeah?

**Review by cerebrl — APPROVED** [review]
Other than the small typo, this looks good.

**Comment by daveadams56** [comment]
Looks good to me

**Pull request merged**